libxw/pyproject.toml

```diff
@@ -0,0 +1,11 @@
+# Configuration file for how to "build" our package.
+#
+# https://www.python.org/dev/peps/pep-0517/
+# https://setuptools.readthedocs.io/en/latest/build_meta.html
```

Comment:
This file is relatively new to the Python ecosystem -- the idea is that `pip` (and other installers, in theory) should be able to "install" packages written with any framework ([setuptools](https://setuptools.pypa.io/en/latest/index.html), [flit](https://flit.pypa.io/en/latest/), [poetry](https://python-poetry.org/), etc.).  The `[build-system]` configuration basically tells `pip` which one of these we want to use, and which third-party dependencies are required to even _try_ installing it.

Reply:
It's pretty confusing and arcane, and luckily you can probably ignore these details.. just mentioning it here in case you're interested :grin: